packages/clarity-js/src/layout/schema.ts

```diff
@@ -0,0 +1,56 @@
+import { Dimension, Metric, Setting } from "@clarity-types/data";
+import { Constant, JsonLD } from "@clarity-types/layout";
+import * as dimension from "@src/data/dimension";
+import * as metric from "@src/data/metric";
+
+/* JSON+LD (Linked Data) Recursive Parser */
+export function ld(json: any): void {
+    for (let key of Object.keys(json)) {
+        let value = json[key];
+        if (key === JsonLD.Type && typeof value === "string") {
+            value = value.toLowerCase();
+            /* Normalizations */
+            value = value.indexOf(JsonLD.Article) >= 0 || value.indexOf(JsonLD.Posting) >= 0 ? JsonLD.Article : value;
```

Comment:
Can you just do === as this indexOf seems like it might over-match.

Reply:
I think that the value will have the article / post id as well which is why we need to do the normalization 

---
In reply to: [581303085](https://github.com/microsoft/clarity/pull/114#discussion_r581303085) [](ancestors = 581303085)